enterprise/app/filter/filter.tsx

```diff
@@ -251,6 +311,9 @@ export default class FilterComponent extends React.Component<FilterProps, State>
       };
     });
 
+    const sortByValue = this.props.search.get(SORT_BY_PARAM_NAME) || "";
+    const sortOrderValue = this.props.search.get(SORT_ORDER_PARAM_NAME) || "desc";
```

Comment:
TS will type these as string, so to get the type checking benefits you can change this to

```suggestion
    const sortByValue: SortBy | "" = (this.props.search.get(SORT_BY_PARAM_NAME) as SortBy) || "";
    const sortOrderValue: SortOrder = (this.props.search.get(SORT_ORDER_PARAM_NAME) as SortOrder) || "desc";
```

Reply:
Whoops, done. I went ahead and added "" as a valid SortBy value too so sortByValue can be just a SortBy here.